vue/sbc-common-components/src/components/BrowserVersionAlert.vue

```diff
@@ -1,85 +1,85 @@
 <template>
-  <sbc-system-banner
-    :show="browserSupported === false"
-    type="warning"
-    :message="alertMessage"
-  ></sbc-system-banner>
+  <v-overlay v-model="browserUnSupported" color="white">
+    <v-dialog v-model="browserUnSupported" max-width="720px" persistent>
+      <v-card>
+        <v-card-title>
+          <div>Unspported Browser </div>
+        </v-card-title>
+        <v-card-text>
+          <p>
+            Internet Explorer 11 is not longer supported as it is coming to end-of-support by Microsoft starting on June 15, 2022.
+            Download one of following browsers for best experience.
+            <a href="https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support" target="_blank">
+              Learn more <v-icon class="text-decoration: none" color="primary" x-small>mdi-open-in-new</v-icon>
```

Comment:
This may be style instead of class, style="text-decoration: none". or define in css and use as  class